### PR TITLE
Add diacritic support to slug generator

### PR DESCRIPTION
**Pull request by erlingur**
Added some basic diacritic support to the slug generator so it works a bit better with foreign languages (Icelandic in my case).

### CODE REVIEW

File: admin/app/webpack/javascripts/components/node-form.es6 [hunk 1]
```diff
@@ -56,9 +56,29 @@ export default Vue.component('node-form', {
   methods: {
     setSlug: function() {
       if (this.node['new_record?']) {
-        this.node.slug = this.node.title.toLowerCase().replace(/[\s\_]/g, '-').replace(/[^\w\-]/g, '');
+        this.node.slug = this.node.title.toLowerCase();
```
Comment (erlingur) [inline admin/app/webpack/javascripts/components/node-form.es6]: Just realized this is all in lowercase so I could dump half of the lines after this line basically. Will do that tonight.

Comment (aaronrussell) [inline admin/app/webpack/javascripts/components/node-form.es6]: 👍 

Also, probably slightly cleaner style to chain it all together, eg:

~~~javascript
this.node.slug = this.node.title.toLowerCase()
                  .replace(/[àáâãäå]/g,"a")
                  .replace(/[éèêëęėē]/g,"e")
                  // etc
                  .replace(/[\s\_]/g, '-').replace(/[^\w\-]/g, '');
~~~

Comment (erlingur) [inline admin/app/webpack/javascripts/components/node-form.es6]: Done! :)

**Comment by erlingur** [comment]
Sweet! 👍

**Comment by mysmallidea** [comment]
I'm not sure if this would work in this case, but Rails built-in `parameterize` takes care of these and other edge cases: 

    >> "àáâãäåéèêëęėēýÿúûüùūíîïìįīóöôòõøōðæþ".parameterize
    => "aaaaaaeeeeeeeyyuuuuuiiiiiiooooooodaeth"

**Pull request merged**